libs/packages/layouts/src/lib/feature/subheader/sds-subheader-wrapper.component.html

```diff
@@ -0,0 +1,30 @@
+<sds-subheader>
+  <!-- =============== BACK BUTTON =============== -->
```

Comment:
There is no way to add additional content or change the content here if needed.

Reply:
Sure - Ill push my changes later today

Reply:
my changes have been added to this PR